### PR TITLE
Update translations_de.yml

### PR DESCRIPTION
**Pull request by MontseVo**

### CODE REVIEW

File: translations_de.yml [hunk 1]
```diff
@@ -107,126 +107,126 @@ stay:
...
     url: 'https://www.visitnavarra.es/en/bardenas-reales'
     src: '../images/bardenas.webp'
     img_alt: 'Castildetierra'
   donosti: 
     title: 'San Sebastián · Donosti'
-    text: '**TODO TRANSLATE**La perla del país vasco, a una hora en coche o <a target="_blank" href="https://www.alsa.es/">en autobús</a> (unos 17€ ida y vuelta). El paseo de la concha es perfecto para hacer un poco de hambre antes de comer unos pintxos en el casco viejo. El peine del viento, el Kursaal, el paseo nuevo, la playa de Zurriola... Nosotros estuvimos en el restaurante <a target="blank" href="https://goo.gl/maps/a7KdV8eHmDK33fNG7">Zazpi</a> y nos encantó.'
+    text: 'Die Perle des Baskenlandes, eine Stunde mit dem Auto oder dem <a target="_blank" href="https://www.alsa.es/">Bus</a> entfernt (ca. 17 € Hin- und Rückfahrt). Ein Spaziergang entlang der Promenade "La Concha" ist perfekt, um " Appetit zu machen" auf ein paar Pintxos in der Altstadt. Außerdem sieht man den Windkamm, den Kursaal, die neue Promenade (paseo nuevo), den Strand von Zurriola... Wir waren im Restaurant <a target="blank" href="https://goo.gl/maps/a7KdV8eHmDK33fNG7">Zazpi</a> und können es nur weiter empfehlen.'
```
Comment (elorz007) [inline translations_de.yml:220]: Gansefüßchen überall

**Review by elorz007 — CHANGES_REQUESTED** [review]

File: translations_de.yml [hunk 2]
```diff
@@ -107,126 +107,126 @@ stay:
...
     url: 'https://www.visitnavarra.es/en/selva-de-irati'
     src: '../images/irati.webp'
-    img_alt: '**TODO TRANSLATE**Selva de irati'
+    img_alt: 'Irati-Wald'
   urederra: 
-    title: '**TODO TRANSLATE**Urbasa y el nacedero del Urederra'
-    text: '**TODO TRANSLATE**Las cascadas más conocidas de Navarra que significa "aguas hermosas" en euskera. Este río con colores turquesa imposibles se puede ver con toda la familia. Ten en cuenta que para visitar el nacedero hace falta pedir cita en <a target="_blank" href="https://urederra.amescoa.com/es/reservas/reservar.html">esta web</a>. Situado en el parque natural de Urbasa, puedes encontrar cerca el mirador del Balcón de Pilatos con unas vistas impresionantes. Cerca está la ciudad de Estella, para ir a tomar algo o comer.'
+    title: '**TODO TRANSLATE**Urbasa und die Quelle der Urederra'
```
[the remark this answers is not in the document]
Comment (MontseVo) [reply translations_de.yml:201]: done

[the remark this answers is not in the document]
Comment (MontseVo) [reply translations_de.yml:201]: i di et

File: translations_de.yml [hunk 3]
```diff
@@ -64,18 +64,18 @@ stay:
       title: 'Alojamientos Olga'
       url: 'https://www.reservaonline.support/apartamentosolga/reservas.html'
     opt3:
-      title: '**TODO TRANSLATE**Buscar apartamentos en Airbnb (evitar aquellos donde ponga cosas de Sanfermines)'
+      title: 'Unterkünfte auf Airbnb suchen (vermeidet dabei die touristischen "Sanfermines"-Unterkünfte)'
```
[the remark this answers is not in the document]
Comment (MontseVo) [reply translations_de.yml:67]: done

File: translations_de.yml [hunk 4]
```diff
@@ -107,126 +107,126 @@ stay:
...
+  summary_places: 'Pamplona ist eine Stadt voller Leben und mit <strong>vielen Sehenswürdigkeiten</strong>. Hier ist eine Liste mit einigen Orten, die ihr euch nicht entgehen lassen solltet, wenn ihr etwas Zeit in Pamplona habt. Alle Sehenswürdigkeiten befinden sich in der Nähe des Stadtzentrums und können zu Fuß besucht werden. Ihr könnt auch einen Blick auf <a target="_black" href="https://www.visitnavarra.es/es/pamplona">web oficial</a> werfen.'
+  summary_food: 'In Pamplona gibt es auch viele <strong>Bars und Restaurants</strong>, wo ihr etwas trinken, ein paar Pintxos essen oder eine Mahlzeit genießen könnt. Wir empfehlen euch auch, einmal eine „Sidrería“ zu besuchen (das Menü sollte aus Chorizo ​​mit Apfelwein, Kabeljau-Omelett, Kalbsrippensteak (Chuletón) und Käse mit Quitte bestehen; alles selbstverstädnlich serviert mit Apfelwein)'
```
[the remark this answers is not in the document]
Comment (MontseVo) [reply translations_de.yml:111]: ignored

File: translations_de.yml [hunk 5]
```diff
@@ -107,126 +107,126 @@ stay:
...
+  title: 'Navarra & Umgebung'
+  summary: 'Navarra bietet eine große Vielfalt an Sehenswürdigkeiten - von üppigen Wälder bis hin zu Wüsten und mitteralterlichen Burgen. Nachfolgend findet ihr einige Ideen für interessante Orte, die ihr weniger als eine Auto-Stunde von Pamplona entfernt besuchen könnt. Für das Meiste braucht ihr ein Auto, aber es ist immer einfach zu parken und fast alle Sehesnwürdigkeiten sind kostenlos (oder es werden nur ein paar Euro Eintritt verlangt). Wenn ihr gerne wandern geht, schaut euch auch mal <a target="_blank" href="https://www.komoot.com/guide/45603/hiking-in-navarra">diese Wanderrouten auf Komootesta</a> an.'
```
[the remark this answers is not in the document]
Comment (MontseVo) [reply translations_de.yml:181]: done

File: translations_de.yml [hunk 6]
```diff
@@ -107,126 +107,126 @@ stay:
...
-    text: '**TODO TRANSLATE**Las cascadas más conocidas de Navarra que significa "aguas hermosas" en euskera. Este río con colores turquesa imposibles se puede ver con toda la familia. Ten en cuenta que para visitar el nacedero hace falta pedir cita en <a target="_blank" href="https://urederra.amescoa.com/es/reservas/reservar.html">esta web</a>. Situado en el parque natural de Urbasa, puedes encontrar cerca el mirador del Balcón de Pilatos con unas vistas impresionantes. Cerca está la ciudad de Estella, para ir a tomar algo o comer.'
+    title: '**TODO TRANSLATE**Urbasa und die Quelle der Urederra'
+    text: 'Die bekanntesten Wasserfälle in Navarra "Urederra", was auf Baskisch "schöne Gewässer" bedeutet. Dieser Fluss mit seinen türkisfarbenen Farben kann mit der ganzen Familie besucht werden. Bitte denkt daran, dass ihr für den Besuch der Quelle einen Termin vereinbaren müsst auf dieser <a target="_blank" href="https://urederra.amescoa.com/es/reservas/reservar.html">Seite</a>. Im Naturschutzgebiet von Urbasa gelegen, befindet sich ganz in der Nähe der Aussichtspunkt "Balcón de Pilatos". Ihr könnet auch in der Stadt "Estella" vorbeischauen, wenn ihr hungrig seid.'
```
[the remark this answers is not in the document]
Comment (MontseVo) [reply translations_de.yml:202]: done

File: translations_de.yml (hunk 1 above)
Comment (MontseVo) [reply translations_de.yml:220]: done

**Review by elorz007 — APPROVED** [review]

**Pull request merged**